packages/grafana-ui/src/components/Forms/RadioButtonList/RadioButtonDot.tsx

```diff
@@ -5,17 +5,27 @@ import { GrafanaTheme2 } from '@grafana/data';
 
 import { useStyles2 } from '../../../themes';
 
-export interface RadioButtonDotProps {
+export interface RadioButtonDotProps<T> {
```

Comment:
Since we don't have any default type defined, will it be required to be specified by the consumers of this type? In that case, can we set a default type?